### PR TITLE
ISPN-5971 Truncate long cache names

### PR DESCRIPTION
**Pull request by tristantarrant**
- Ellipsize the cache card names
- Fix padding between container status label and name
- Use default cursor on trait icons

### CODE REVIEW

File: src/main/webapp/cluster-view/cluster-view.less [hunk 1]
```diff
@@ -29,18 +18,3 @@
   float: right;
   border: 1px solid #D8D8D8;
 }
-.card-link {
```
Comment (vblagoje) [inline src/main/webapp/cluster-view/cluster-view.less]: @tristantarrant this was a request from Rome f2f (click anywhere on the card to activate link)

**Comment by vblagoje** [comment]
@tristantarrant I could not get ng-click to work on the card itself. I suggest we keep the current CSS that enables this functionality. If we find a way to do it somehow differently we can always amend the current solution. LMK

**Comment by tristantarrant** [comment]
@vblagoje I have fixed it by wrapping the card divs in a's. I've also removed the version from the login page

**Comment by vblagoje** [comment]
Integrated 0f030e36eda4874f55f4b677578c5205e833eaee Thanks @tristantarrant

**Pull request closed without merging**